plugins/techdocs/src/Router.tsx

```diff
@@ -69,7 +69,9 @@ export const EmbeddedDocsRouter = (props: PropsWithChildren<{}>) => {
 
   return (
     <Routes>
-      <Route element={<EntityPageDocs entity={entity} />}>{children}</Route>
+      <Route path="*" element={<EntityPageDocs entity={entity} />}>
```

Comment:
Looked back at the commits and it seemed to have been changed here https://github.com/backstage/backstage/commit/754409bef50a8d32905c8e3bc3862268469d88cd. Before the path was `/*` not sure if that matters as the parent route is `/docs` https://github.com/backstage/backstage/blob/32b44f571d3839794a22ff726d5649a1c065e610/packages/create-app/templates/default-app/packages/app/src/components/catalog/EntityPage.tsx#L169

Reply:
@emmaindal added `/*` in this pull request:
https://github.com/backstage/backstage/pull/11546